src/index.ts

```diff
@@ -48,3 +49,5 @@ export const edkt = {
   run,
   getMatchedAudiences,
 };
+
+export const audienceDefinitions = audienceMap;
```

Comment:
Will this tree-shake? If not then this will not work - the idea is that each consumer can only pull in wanted audiences.